core/src/test/java/org/apache/calcite/test/TableFunctionTest.java

```diff
@@ -408,6 +408,23 @@ private Connection getConnectionWithMultiplyFunction()
     with().query(q).returnsUnordered("C=7");
   }
 
+  @Test public void testTableFunctionInSelect() {
+    final String q = "select * from ("
+        + "select f0, \"s\".\"fibonacci2\"(20) as (c)"
+        +    " from (select 1 as f0)"
+        + ")";
```

Comment:
In hive, I remember that you cannot select UDTF and other column in one select. Please correct me if I am wrong.

Reply:
Yes, I support this feature by the way.

Reply:
I think we'd better do exactly the same as Hive does.

Reply:
Well, I think it is a useful feature for people to write udtf with other column in select statement.

Reply:
> I think we'd better do exactly the same as Hive does.

Not necessarily. Hive does some pretty dumb things, and I don't want to let that stupidity into Calcite. If Hive differs from standard SQL, we need to talk about it.